packages/strapi-plugin-content-manager/admin/src/containers/EditViewDataManagerProvider/index.js

```diff
@@ -251,7 +252,7 @@ const EditViewDataManagerProvider = ({
       abortController.abort();
     };
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [id, slug, isLoadingForPermissions]);
+  }, [id, origin, slug, isCreatingEntry, isLoadingForPermissions]);
```

Comment:
Adding isCreating entry as a dependency causes requests to being sent multiple times especially when the view changes from creation to edition.


Here, you are facing the issue where the hook as too many dependencies in his dependency array, which is causing unwanted behaviour (in this case requests being fired multiple time).

We also faced the issue with the D&P feature where adding `isCreating` entry to the dependencies array caused the same issue. However in our case removing it was ok as we switch components and add a loading state between the creation and edition.

I am planning to refactor the hook especially since we will need this for the internationalisation feature but I am afraid you might need to do it yourself if we were to add the clone feature to the CM.

Feel free to DM on Slack so we can further discuss about it.